docarray/computation/numpy_backend.py

```diff
@@ -44,6 +44,11 @@ def stack(
     def n_dim(array: 'np.ndarray') -> int:
         return array.ndim
 
+    @staticmethod
+    def none_value() -> Any:
+        """Provide a compatible value that represents None in torch."""
+        return None
```

Comment:
Why not `np.array(float('nan'))`?

Reply:
Numpy do handle None properly. Though we can decide to use Nan as well to do the same as in torch not sure though that it makes sense